.eslintrc.js

```diff
@@ -307,6 +315,19 @@ module.exports = {
         'Key': false,
       },
     },
+    {
+      'files': [
```

Comment:
I did this naïvely by moving the `presubmit` logic into its own ruleset override. Happy to distribute this into other existing rules.

Reply:
Moving this here addresses my previous concern of having this hidden inside a custom line rule implementation file.